polaris.shopify.com/pages/sandbox/index.tsx

```diff
@@ -18,29 +17,12 @@ export const getServerSideProps: GetServerSideProps = async ({query}) => {
   };
 };
 
-const MS_DELAY_BEFORE_SHOW_ONBOARDING = 500;
-
 export default function Sandbox({
   initialSearchParams,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
   const iframeRef = useRef<HTMLIFrameElement>(null);
   const router = useRouter();
   const searchValue = useRef('');
-  const [isHelpOpen, setHelpIsOpen] = useState(false);
```

Comment:
All this has been moved into polaris.shopify.com/src/components/SandboxHeader/SandboxHeader.tsx